migration/README.md

```diff
@@ -0,0 +1,19 @@
+# unified-codes : data migration
```

Comment:
I think it might be better to make a separate private repo for the migration? Might be best to not make the data public?

Reply:
yeah. will remove this folder - speaks to above points about not having data in repo as well, though the migration code does need to be shared somewhere, and I think versioning is helpful for any code files.